.github/ISSUE_TEMPLATE/z-learn-feedback-tool.yml

```diff
@@ -0,0 +1,66 @@
+name: Learn feedback control
+description: |
+  ⛔ This template is hooked into the feedback control on the bottom of every page on the live site. It automatically fills in several fields for you. Don't use for other purposes. ⛔
+body:
+  - type: markdown
+    attributes:
+      value: "## Issue information"
+  - type: markdown
+    attributes:
+      value: Select the issue type, and describe the issue in the text box below. Add as much detail as needed to help us resolve the issue.
+  - type: dropdown
+    id: issue-type
+    attributes:
+      label: Type of issue
+      options:
+        - Typo
+        - Code doesn't work
+        - Missing information
+        - Outdated article
+        - Other (describe below)
```

Comment:
Can we modify/add categories here or are we trying to standardize with other teams? what's the guidance here?